### DISCUSSION TITLE
text overflow issue on commiter UI

### DISCUSSION THREAD

**Discussion by wooseok123** [community/community#75448] [Pull Requests] [answered]
### Select Topic Area

Bug

### Body

![Screenshot_20231114_025125_Samsung Internet](https://github.com/community/community/assets/59460718/9159fab7-92c7-4ca3-b6ea-9de1607b2dad)

When i click the commits to get a closer look on the code, the text that describes the committer is over the line as above.

Maybe it needs to add text-overflow : ellipsis to hide overflowed texts. Or setting the area horizontally draggable would be also great option.(maybe overflow-x : auto?)

**Comment by HenSquared** [comment, marked as answer]
@wooseok123 , thanks a lot for your report. I've categorised this discussion to 'Pull Requests' since this is related to the PR screen on web. 

I have opened an internal issue regarding this issue as well in the supporting team's repository and included a link to the commit in question. Thanks a lot for the feedback. :)
